MapboxNavigationTests/LaneTests.swift

```diff
@@ -27,33 +27,33 @@ class LaneTests: FBSnapshotTestCase {
         steps = routeController.routeProgress.currentLeg.steps
         routeProgress = routeController.routeProgress
     }
-    
+
     func assertLanes(stepIndex: Array<RouteStep>.Index) {
         let rect = CGRect(origin: .zero, size: .iPhone6Plus)
         let navigationView = NavigationView(frame: rect)
-        
+
         routeController.advanceStepIndex(to: stepIndex)
-        
+
         navigationView.lanesView.update(for: routeController.routeProgress.currentLegProgress.currentStepProgress.currentVisualInstruction)
         navigationView.lanesView.show(animated: false)
-        
+
         FBSnapshotVerifyView(navigationView.lanesView)
     }
-    
+
     func testStraightRight() {
         assertLanes(stepIndex: 0)
     }
-    
+
     func testRightRight() {
         assertLanes(stepIndex: 1)
     }
-    
+
     func testSlightRight() {
         let view = LaneView(frame: CGRect(origin: .zero, size: CGSize(width: 30, height: 30)))
         view.backgroundColor = .white
         view.lane = Lane(indications: [.slightRight])
         view.maneuverDirection = .slightRight
         view.isValid = true
-        FBSnapshotVerifyView(view, suffixes: ["_64"])
```

Comment:
Again, this needs to stay. the `suffixes: ["_64"])` argument prevents FBSnapshotTests from looking in `ReferenceImages_32`, which we don't have. 

Really, all verify calls should be ` FBSnapshotVerifyView(view, suffixes: ["_64"])`.

It basically stops the testing engine from making extra noise on failures.

Reply:
For Posterity:

Q:
> Can you expand a little bit? We only test on iPhone 6 Plus iOS 11 and 12. Why do we need the 64bit suffix?

A: 
> because by default `FBSnapshotView` looks at `ReferenceImages_32` as a fallback if it can't find the image in `ReferenceImages_64`. Since we don't do 32-bit builds, this does nothing but emit an extra, unnecessary error, which adds noise to the log.

Reply:
FWIW, I'd prefer extra noise on failure than extra noise in tests. Especially now when we have `./scripts/generate_snapshots.sh` so we don't need to manually find the missing snapshots.

Reply:
the "extra noise" conveys exactly 0 extra useful information, because it's always preceded by an error for the `ReferenceImages_64` lookup.

It's just extra cruft that impedes searching through CI logs, it is 100% unnecessary. 

Having things fail loudly is nice, but I'd rather not have any noise in the log at all.

Reply:
I get that the information is not useful, but I'd argue that the suffix parameter isn't useful either. Two wrongs don't make a right 😇 If you see a missing snapshot in the logs, 32 or 64 doesn't matter, just run `./scripts/generate_snapshots.sh` and the error goes away.

Reply:
@frederoni I feel quite strongly about this, as those errors were the source of some major frustration for me not too long ago. I understand the annoyance of having to write an extra argument on the verify call, so I went ahead and made a wrapper call `verify(someView)` that seems to me to be the best of both worlds: It's easier to write then `FBSnapshotVerifyView(someView)`, there are no extra arguments to worry about, and it adds the `suffixes` argument for you.

The change is [here.](
https://github.com/mapbox/mapbox-navigation-ios/pull/1709/commits/aee628955df40b60bd2f45c1c0604a54c5d4cff) What do you think?